identity/ffi/src/lib.rs

```diff
@@ -1,4 +1,7 @@
 pub use sunshine_ffi_utils as utils;
+
+#[cfg(feature = "faucet")]
```

Comment:
Can't we remove this all together?

Reply:
we don't want a breaking change for https://github.com/sunshine-protocol/sunshine-identity-ui/issues/7

Reply:
Why not? We'll be having demo 2.0 soon (before end of week?)